packages/opentelemetry-node-tracer/test/NodeTracer.test.ts

```diff
@@ -53,20 +71,49 @@ describe('NodeTracer', () => {
         scopeManager: new AsyncHooksScopeManager(),
       });
       assert.ok(tracer instanceof NodeTracer);
+      tracer.stop();
     });
 
     it('should construct an instance with logger', () => {
       const tracer = new NodeTracer({
         logger: new NoopLogger(),
       });
       assert.ok(tracer instanceof NodeTracer);
+      tracer.stop();
     });
 
     it('should construct an instance with sampler', () => {
       const tracer = new NodeTracer({
         sampler: ALWAYS_SAMPLER,
       });
       assert.ok(tracer instanceof NodeTracer);
+      tracer.stop();
+    });
+
+    it('should load user configured plugins', () => {
+      const tracer = new NodeTracer({
+        logger: new NoopLogger(),
+        plugins: {
+          'simple-module': {
+            enabled: true,
+            path: '@opentelemetry/plugin-simple-module',
+          },
+          'supported-module': {
+            enabled: true,
+            path: '@opentelemetry/plugin-supported-module',
+            enhancedDatabaseReporting: false,
+            ignoreMethods: [],
+            ignoreUrls: [],
+          },
+        },
+      });
+      const pluginLoader = tracer['_pluginLoader'];
+      assert.strictEqual(pluginLoader['_plugins'].length, 0);
+      require('simple-module');
+      assert.strictEqual(pluginLoader['_plugins'].length, 1);
+      require('supported-module');
+      assert.strictEqual(pluginLoader['_plugins'].length, 2);
+      tracer.stop();
```

Comment:
Should this be in some kind of `afterEach`?

Reply:
Done.